geth/jail/handlers.go

```diff
@@ -66,23 +66,54 @@ func registerHandlers(jail *Jail, cell common.JailCell, chatID string) error {
 	return nil
 }
 
+// makeAsyncSendHandler returns jeth.sendAsync() handler.
+func makeAsyncSendHandler(jail *Jail, cellInt common.JailCell) func(call otto.FunctionCall) otto.Value {
+	// FIXME(tiabc): Get rid of this.
+	cell := cellInt.(*Cell)
+	return func(call otto.FunctionCall) otto.Value {
+		go func() {
+			response := jail.Send(call, cell.VM)
```

Comment:
Do we have tests ensuring we are not loosing references anymore?

Reply:
We've never lost references. These were just race conditions. But we have one decent test anyway: https://github.com/status-im/status-go/blob/6800cf81d10c9d50ca92bc2d2b718097d0c57cef/geth/jail/jail_test.go#L128

Reply:
Alright